db/version_set.cc

```diff
@@ -2510,17 +2511,31 @@ void VersionStorageInfo::CalculateBaseBytes(const ImmutableCFOptions& ioptions,
         }
       }
 
+      const uint64_t kMinNonEmptyLevelSize = static_cast<uint64_t>(
+          base_bytes_max / options.max_bytes_for_level_multiplier);
```

Comment:
This is only the naming for "Variables declared constexpr or const, and whose value is fixed for the duration of the program": https://google.github.io/styleguide/cppguide.html#Constant_Names . This is probably doesn't qualify.